src/index.ts

```diff
@@ -1212,23 +1319,25 @@ class DevServer {
       return;
     }
 
-    const debigPid = await this.waitForNodeChildProcess(parseInt(match[1]));
+    const debugPid = await this.waitForNodeChildProcess(parseInt(match[1]));
 
-    this.log.box(`Debugger is now available on process id ${debigPid}`);
+    this.log.box(`Debugger is now available on process id ${debugPid}`);
   }
 
   async setupDevServer(
     adminPort: number,
     dependencies: DependencyVersions,
+    entrypoint: string,
     backupFile: string | undefined,
     useSymlinks: boolean,
   ): Promise<void> {
-    await this.buildLocalAdapter();
+    // await this.buildLocalAdapter();
```

Comment:
nit: personally I am not a fan of out commented code, as we have git history to know how it looked like a while ago

Reply:
I may have forgotten to clean this up 😇